tensornetwork/network_operations.py

```diff
@@ -92,14 +93,14 @@ def transpose(node: BaseNode,
 
 def copy(nodes: Iterable[BaseNode],
          conjugate: bool = False) -> Tuple[dict, dict]:
-  """
+  """Return a copy of the given nodes.
```

Comment:
I would be more specific in this docstring. Maybe say "Copy the given nodes and their edges, returning a dictionary linking originals to copies."

Reply:
Done.